conn.go

```diff
@@ -739,6 +741,12 @@ func (c *Conn) handleBdat(arg string) {
 		}()
 	}
 
+	prevLineLimit := c.lineLimitReader.LineLimit
+	c.lineLimitReader.LineLimit = 0
+	recover := func() {
+		c.lineLimitReader.LineLimit = prevLineLimit
+	}
+	defer recover()
```

Comment:
`recover` means something else for Go, it refers to handling a panic. This is confusing, because we don't handle a panic here.

Why do we need to have an intermediary variable here?

Reply:
fix function name.